src/widgets/components/tooltip_widget.py

```diff
@@ -0,0 +1,21 @@
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QIcon
+from PyQt5.QtWidgets import QWidget, QHBoxLayout, QLabel
+
+
+class QLabelWithTooltip(QWidget):
+    def __init__(self, label: str, description: str = ""):
+        super().__init__()
+
+        self.layout = QHBoxLayout(self)
+        self.layout.setContentsMargins(0, 0, 0, 0)
+
+        self.label = QLabel(label)
+        self.layout.addWidget(self.label, alignment=Qt.AlignVCenter)
+
+        if description:
+            self.icon = QIcon.fromTheme("dialog-information").pixmap(14)
```

Comment:
this works only on Linux :(

on mac I did something like this:
```
self.icon = self.style().standardIcon(QStyle.StandardPixmap.SP_FileDialogInfoView).pixmap(14)
```
but this Qt icon is quite ugly. We could find/make a better one and just use it instead of relying on Qt or system. I can look into this later